stock_by_warehouse_sale/i18n/es.po

```diff
@@ -0,0 +1,42 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* stock_by_warehouse_sale
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 11.0+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-04-04 20:34+0000\n"
+"PO-Revision-Date: 2018-04-04 20:34+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: stock_by_warehouse_sale
+#: model:ir.model,name:stock_by_warehouse_sale.model_sale_order_line
+msgid "Sales Order Line"
+msgstr "Línea de pedido de venta"
+
+#. module: stock_by_warehouse_sale
+#: model_terms:ir.ui.view,arch_db:stock_by_warehouse_sale.sale_order_form_view
+msgid "The saleable stock in:"
+msgstr "La existencia vendible en:"
+
+#. module: stock_by_warehouse_sale
```

Comment:
@hugho-ad  Just missing the removal of this translation

Reply:
thanks!